src/runtime-rs/crates/hypervisor/src/device/mod.rs

```diff
@@ -24,26 +24,19 @@ pub enum DeviceConfig {
     BlockCfg(BlockConfig),
     VhostUserBlkCfg(VhostUserConfig),
     NetworkCfg(NetworkConfig),
-    ShareFsCfg(ShareFsDeviceConfig),
+    ShareFsCfg(ShareFsConfig),
     VfioCfg(VfioConfig),
-    ShareFsMountCfg(ShareFsMountConfig),
     VsockCfg(VsockConfig),
     HybridVsockCfg(HybridVsockConfig),
 }
 
-#[derive(Debug)]
-pub enum UpdateDeviceConfig {
-    ShareFsCfg(ShareFsMountConfig),
-}
-
```

Comment:
Sorry @gkurz, I have indeed removed the UpdateDeviceConfig, But maybe it should placed in the previous commit.

Reply:
Yes. Thanks !